buildchow/templates/aggregate-site-template.yaml

```diff
@@ -10,11 +10,11 @@
 
     node: "{nodes}"
     logrotate:
-      daysToKeep: 5
```

Comment:
tip: if we are going to use the exact same logrotate for all (makes sense) then that could be done as a macro.